prysm/host-path/README.md

```diff
@@ -15,13 +15,17 @@ This example demonstrates how to run one validator client and one beacon node us
     git clone https://github.com/lumostone/eth2xk8s.git
     ```
 
-2. Create the data folders for beacon node and validator on the host machine. For example:
+2. Create the data folders for beacon node and validator and change the ownership. For example:
```

Comment:
We should change the ownership after we import the validator account, which means this should be separated into a new step after "importing validator accounts".

Reply:
It's the same for https://github.com/lumostone/eth2xk8s/tree/master/prysm/nfs. Can you also change that file?